docs/guides/4-custom-rulesets.md

```diff
@@ -86,7 +86,7 @@ responses:
     foo: bar
 ```
 
-## Extending Rules
+## Extending Ruleset
```

Comment:
>The paragraph describe how to extend a rulesets not a rule.

I'm not sure about that change. Indeed the document relates to rulesets, but my understanding of that paragraph is that is describes how to override and replace an rule that already exists in one of the base ruleset.

Aside from that, maybe the title is a bit misleading as we don't really extend a rule, but rather redefine it.

So maybe 

```suggestion
## Redefining Rules
```

@philsturgeon Thoughts?

Reply:
Replacing Rules? Then it’s consistent with language in the paragraph you’re suggesting.